woocommerce/woocommerce
# Move "Grow your business" to "Things to do next" task list

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes https://github.com/woocommerce/woocommerce/issues/50486. 

This moves the "Grow your business" task to the "Things to do next" task list. With Launch your store now released, it is more appropriate for new merchants to focus on growing their store after the store is launched.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Go to WooCommerce > Home
2. See that "Grow your business" is in the "Things to do next" task list.
3. Test around the "Grow your business" task, e.g. install a recommended extenson.
4. See that the home badge reflects the amount of unfinished tasks in the main setup task list.

<img width="1792" alt="grow yo business" src="https://github.com/user-attachments/assets/f108e1cf-4ca6-4925-b078-fa3f8f456dd4">





### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
